kubernetes/structure_service_spec.go

```diff
@@ -138,6 +177,30 @@ func expandIPFamilies(l []interface{}) []v1.IPFamily {
 	return obj
 }
 
+func expandSessionAffinityConfigClientIP(l []interface{}) *v1.ClientIPConfig {
+	var to int32
```

Comment:
you need to add a guard here to check that the length isn't 0 and the first item isn't null, like this: 

```golang
if len(l) == 0 || l[0] == nil {
	return &v1.ClientIPConfig{}
}
```

It's an edge case but I was able to crash the provider by specifying this block but omitting `timeout_seconds`.

Reply:
Good catch, thank you!